scss/partials/_cards.scss

```diff
@@ -63,13 +69,13 @@
     max-width: 100%;
   }
 
-  &__subtitle {
+  &__lower &__subtitle {
```

Comment:
This selector feels not BEM, since it results in two chained elements.

Reply:
I was trying to add more specificity score to the element to overwrite the default `<p>` tag color. Should I think of another work around @fienen?

Reply:
I'd just toss on a new modifier instead. `--somecolor` or something like that.

Reply:
For what it's worth - this is another way to increase specificity. 
https://css-tricks.com/the-sass-ampersand/#doubling-up-specificity

Reply:
think I fixed it right ... Could you guys double check for me please?